_targets.R

```diff
@@ -110,20 +109,18 @@ list(
     command = rbind(old_tidy_field_notes, mWater_field_notes)
   ),
 
-  # load incoming API data
+  # Load incoming API data:
   # to do (j): try to convert this into a tar_file_read() function
   tar_target(
     name = incoming_data_collated_csvs,
     command =  {
       incoming_data_csvs_upload
       munge_api_data(api_path = "data/api/incoming_api_data/")
-      },
-    packages = "tidyverse"
+    },
+    packages = c("tidyverse", "data.table")
   ),
 
-  # format data
-
-  ## generate a site parameter combination list
+  # Generate a site parameter combination list to iterate over:
```

Comment:
praise: I like these more descriptive comments, something I will try to be more cognizant of as well.